examples/src/examples/animation/blend-trees-1d.example.mjs

```diff
@@ -2,10 +2,8 @@ import * as pc from 'playcanvas';
 import { data } from 'examples/observer';
 import { deviceType, rootPath } from 'examples/utils';
 
-const canvas = document.getElementById('application-canvas');
-if (!(canvas instanceof HTMLCanvasElement)) {
-    throw new Error('No canvas found');
-}
+const canvas = /** @type {HTMLCanvasElement} */ (document.getElementById('application-canvas'));
```

Comment:
Casting is not knowing... fake type safety that breaks at any refactor

Reply:
application-canvas is hard coded into the html file so its guaranteed.